src/libraries/System.ComponentModel.TypeConverter/tests/TrimmingTests/SampleStream.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace TrimmingTests
+{
+    internal static class SampleStream
```

Comment:
Maybe this file should be named `DesigntimeLicenseContextSerialization_Stream.cs` to give it a more specific name?